Update README.md (#3529)
amplication/amplication: README.md

Amplication is currently in Beta version. This means that we are still working on essential features like production-ready hosting, migrations, and stability to our console.

> How does it affect you? Well... mostly it doesn't. Every app generated using Amplication platform contains popular, documented, secured, and supported production-ready open-source components & packages. Your app is stable, scalable, and production-ready you can deploy and rely on. You can read more about the generated app and its stack here  [Get Started](https://docs.amplication.com/docs/getting-started)

Reviewer: ```suggestion
How does it affect you? Well... mostly it doesn't. Every app generated using Amplication platform contains popular, documented, secured, and supported production-ready open-source components & packages. Your app is stable, scalable, and production-ready you can deploy and rely on. You can read more about the generated app and its stack in our [Getting Started documentation](https://docs.amplication.com/docs/getting-started).
```
